### ISSUE TITLE
bug: CtLambda:getMethod introduces a regression

### ISSUE THREAD

**Issue by surli** [INRIA/spoon#1176]
The PR #1164 seems to introduce a regression when using Spoon on RxJava: 
```
spoon.SpoonException: The lambda can be based on interface, which has only one method. But io.reactivex.NbpObservable$NbpOperator has at least two: <T extends java.lang.Object> java.util.function.Function identity() and java.lang.Object apply(java.lang.Object)
```

See the full log there: https://ci.inria.fr/sos/job/RxJava/419/console

**Comment by monperrus** [comment]
fixed by b63ac185bb5c83b60f1c1bdbc10aa9b4004ae3a6

**Comment by monperrus** [comment]
BTW: Nopol shoud be able to fix this one.
